src/main/io/gps.c

```diff
@@ -1476,15 +1993,15 @@ typedef struct ubxNavSvinfo_s {
     uint8_t numCh;              // Number of channels
     uint8_t globalFlags;        // Bitmask, Chip hardware generation 0:Antaris, 1:u-blox 5, 2:u-blox 6
     uint16_t reserved2;         // Reserved
-    ubxNavSvinfoChannel_t channel[GPS_SV_MAXSATS_M8N];         // 32 satellites * 12 byte
+    ubxNavSvinfoChannel_t channel[GPS_SV_MAXSATS_M8N];         // 48 satellites * 12 byte
```

Comment:
Isn't it back to 32 ?

Reply:
Yes it should be 32, I didn't update that comment, I'll fix that